vm-setup/roles/v1aX_integration_test/tasks/move_back.yml

```diff
@@ -7,7 +7,7 @@
       kind: Deployment
       state: absent
       namespace: "{{ NAMESPACE }}"
-      kubeconfig: "/tmp/kubeconfig-{{ CLUSTER_NAME }}.yaml"
+      kubeconfig: "{{ TARGET_KUBECONFIG }}"
```

Comment:
If `move_back`  is called in local deployment separately after `move` was done separately, would the variable `{{ TARGET_KUBECONFIG }}` have the correct value which was registered during `move` step?